tests/impl.cpp

```diff
@@ -1857,6 +1859,14 @@ bool test_mm_cvtps_epi32(const float _a[4])
     return validateInt32(ret, trun[0], trun[1], trun[2], trun[3]);
 }
 
+bool test_mm_testz_si128(const int32_t *_a, const int32_t *_b)
+{
+    __m128i a = _mm_load_si128((const __m128i *) _a);
+    __m128i b = _mm_load_si128((const __m128i *) _b);
+    bool testz = validate128(a, b);
+    return _mm_testz_si128(a, b) == testz;
```

Comment:
I don't understand the comparison here.

Consider the following example:
||a=0b00...00, b=0b00...00|a=0b0101...01, b=0b1010...10|
|:-:|:-:|:-:|
|validate128|true|false|
|_mm_testz_si128|ZF=1|ZF=1|